core/domain/fs_domain.py

```diff
@@ -85,7 +94,9 @@ def __init__(self, entity_name, entity_id):
         self._validate_entity_parameters(entity_name, entity_id)
         self._assets_path = '%s/%s/assets' % (entity_name, entity_id)
 
-    def _validate_entity_parameters(self, entity_name, entity_id):
+    def _validate_entity_parameters(
+            self, entity_name: str, entity_id: str
+        ) -> None:
```

Comment:
```suggestion
    def _validate_entity_parameters(
        self, entity_name: str, entity_id: str
    ) -> None:
```

please make sure to address my comments correctly.

Reply:
ok